src/main/java/uk/gov/companieshouse/charges/data/config/ExceptionHandlerConfig.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.charges.data.config;
+
+import java.time.LocalDateTime;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+@ControllerAdvice
+public class ExceptionHandlerConfig {
+
+    /**
+     * Runtime exception handler.
+     *
+     * @param ex      exception to handle.
+     * @param request request.
+     * @return error response to return.
+     */
+    @ExceptionHandler(value = {Exception.class})
+    public ResponseEntity<Object> handleException(Exception ex, WebRequest request) {
+
+        Map<String, Object> responseBody = new LinkedHashMap<>();
+        responseBody.put("timestamp", LocalDateTime.now());
+        responseBody.put("message", "There is issue completing the request.");
+        responseBody.put("correlationId", UUID.randomUUID());
```

Comment:
To keep it simple for support team to investigate in future, we can just keep first 8 characters of UUID, and all upper case if you agree

Reply:
Agree, we need to shorten the correlationId to make it more readable when going through logs.